52.md

```diff
@@ -0,0 +1,142 @@
+* `start` (required) inclusive start Unix timestamp in seconds. Must be less than `end`, if it exists.
+* `end` (optional) exclusive end Unix timestamp in seconds
+* `location` (optional) location of the calendar event. e.g. address, GPS coordinates, meeting room name, link to video call
+* `g` (optional) [geohash](https://en.wikipedia.org/wiki/Geohash) to associate calendar event with a searchable physical location as suggested as an example by [NIP-12](12.md)
+* `p` (optional, repeated) 32-bytes hex pubkey of a participant
+* `t` (optional, repeated) hashtag to categorize calendar event
+* `r` (optional, repeated) references / links to web pages, documents, video calls, recorded videos, etc.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>,
+  "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
+  "created_at": <Unix timestamp in seconds>,
+  "kind": "31923",
+  "tags": [
+    ["d", "<UUID>"],
+
+    ["summary", "<summary of calendar event>"],
+    ["description", "<description of calendar event>"],
+
+    // Timestamps
+    ["start", "<Unix timestamp in seconds>"],
+    ["end", "<Unix timestamp in seconds>"],
+
+    // Location
+    ["location", "<location>"],
+    ["g", "<geohash>"],
+
+    // Participants
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>"],
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>"],
+
+    // Hashtags
+    ["t", "<tag>"],
+    ["t", "<tag>"],
+
+    // Reference links
+    ["r", "<url>"],
+    ["r", "<url>"]
+  ]
+}
+```
+
+Calendar events are not required to be part of a [calendar](#calendar).
+
+## Calendar Event RSVP
+
+A calendar event RSVP is a response to a calendar event to indicate a user's attendance intention.
+
+If a calendar event tags a pubkey, that can be interpreted as the calendar event creator inviting that user to attend. Clients MAY choose to prompt the user to RSVP for the calendar event.
+
+Any user may RSVP, even if they were not tagged on the calendar event. Clients MAY choose to prompt the calendar event creator to invite the user who RSVP'd. Clients also MAY choose to ignore these RSVPs.
+
+This NIP is intentionally not defining who is authorized to attend a calendar event if the user who RSVP'd has not been tagged. It is up to the calendar event creator to determine the semantics.
+
+This NIP is also intentionally not defining what happens if a calendar event changes after an RSVP is submitted.
+
+### Format
+
+The format uses a parameterized replaceable event kind `31924` with a list of tags as described below:
+* `a` (required) reference tag to kind `31923` calendar event being responded to per [NIP-33](33.md)
+* `d` (required) universally unique identifier. Generated by the client creating the calendar event RSVP.
+* `t` (required) attendance status. It can be one of the following values: `yes`, `no`, `maybe`
```

Comment:
Changing `yes`, `no`, `maybe` to `accepted`, `declined`, `tentative` respectively to follow the iCal spec naming more closely.